resources/language/English/strings.po

```diff
@@ -111,3 +143,11 @@ msgstr ""
 msgctxt "#32035"
 msgid "It is recommended that you to upgrade to a newer version."
 msgstr ""
+
+msgctxt "#32037"
+msgid "Do you want checking updates?"
```

Comment:
This doesn't seem to be used

Reply:
J'ai supprimé le numéro inutile et renuméroté le formulaire (view new commit)

Reply:
In English
I deleted the useless number and renumbered the form